docs/docs/identity-providers/okta.md

```diff
@@ -4,11 +4,13 @@ lang: en-US
 sidebarDepth: 0
 meta:
   - name: keywords
-    content: okta oidc, identity provider, idp
+    content: okta oidc, identity provider, idp, context, context-aware,
 ---
 
 # Okta
 
+Okta is a popular identity provider used by businesses of all sizes. Integrating Pomerium with Okta allows you to use the identity they provide to apply context-driven policies to your infrastructure.
```

Comment:
It wasn't totally clear to me who is providing the context-driven policies here. Maybe can be reworded to be more clear?